v2/googlecloud-to-googlecloud/docs/BigtableChangeStreamsToPubSub/README.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Bigtable Change Streams To Pub/Sub Dataflow Template
+NOTE: This template is currently unreleased. If you wish to use it now, you
+will need to follow the steps outlined below to add it to and run it from
+your own Google Cloud project. Make sure to be in the /v2 directory.
+
+The [BigtableChangeStreamsToPubSub](src/main/java/com/google/cloud/teleport/v2/templates/bigtablechangestreamstopubsub/BigtableChangeStreamsToPubSub.java)
+pipeline reads messages from Cloud Bigtable Change Streams and publish them through Pub/Sub.
+
+Change log schema in different formats such as JSON, AVRO and PROTOCOL_BUFFERS are available in .avsc and .proto files resources directory.
+And the change log schema is defined as follows:
+
+| Field name      | Protobuf Type | Avro Type      | JSON Type | Nullable? | Description                                                                                                                                  |
+|-----------------|------|----------------|-----------|-----------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| rowKey          | STRING | STRING         | STRING    | N         | Bigtable row key                                                                                                                             |
+| modType         | STRING | STRING         | STRING    | N         | Modification type: {SET_CELL, DELETE_CELLS, DELETE_FAMILY}. DeleteFromRow mutation is converted into a series of DELETE_FROM_FAMILY entries. |
+| isGC            | BOOL | BOOLEAN        | BOOL      | Y         | TRUE indicates that mutation was made by garbage collection in CBT                                                                           |
+| tiebreaker*     | INT32 | INT            | INTEGER   | N         | CBT tie-breaker value. Used for conflict resolution if two mutations are committed at the sametime.                                          | 
+| commitTimestamp | TIMESTAMP | TIMESTAMP-MILLIS | STRING    | N         | Time when CBT wrote this mutation to a tablet                                                                                                |
+| columnFamily    | STRING | STRING         | STRING    | N         | CBT column family name                                                                                                                       |
+| column          | STRING | STRING         | STRING    | Y         | CBT column qualifier                                                                                                                         |
```

Comment:
This README should have lines about these newly added fields:

  required string sourceInstance = 12;
  required string sourceCluster = 13;
  required string sourceTable = 14;